script/upload

```diff
@@ -123,13 +123,36 @@ create_release () {
     sed -e 's/{.*}//g'
 }
 
+# Update the draft release with a new body and print the upload URL for release assets to the
+# standard output.  A release with the given version must already exist.
+patch_release () {
+  local version="$1"
+  local bodyfile="$2"
+
+  # Find the URL of this release.
+  local url=$(curl https://api.github.com/repos/$REPO/releases | \
+    jq -r '.[] | select(.name == "'"$version"'") | .url')
+
+  [ -n "$url" ]
```

Comment:
Since we're running with `set -e`, the script aborts if any command (other than the condition of a conditional) exits unsuccessfully.  So in this case, the script would exit if the test were false.

This is more of a sanity check (equivalent to an assert) than anything else: we want to be sure that there is some release that we're operating on, or we're going to have a bad time.

Reply:
I'll add an abort here so that there's a useful error message.